tap_twilio/streams.py

```diff
@@ -155,8 +159,7 @@
                 'key_properties': ['sid'],
                 'replication_method': 'INCREMENTAL',  # Filter query
                 'replication_keys': ['end_time'],
-                'bookmark_query_field_from': 'EndTime>',  # Daily
-                'bookmark_query_field_to': 'EndTime<',
+                'bookmark_query_field_from': 'StartTime>',  # Daily
```

Comment:
for **calls** stream you have changed the query filter from `EndTime` to `StartTime`, but the bookmark key is still the `end_time`
shouldn't the replication key also be the `start_time` (because we will be storing the end time as the bookmark, but using it to filter by start_time) this might result in creating a gap window between two syncs.
is there any specific reason to not change the `replication_key`?
i see `date_created` / `date_updated` available in the schema, which should be ideally used as replication_keys, can we filter by these properties?

Reply:
Yes, you're right. query_param should be `EndTime` and replication_key should be `date_updated`. made changes